lib/Backend/LinearScan.cpp

```diff
@@ -4974,16 +4968,6 @@ LinearScan::GeneratorBailIn::~GeneratorBailIn()
     JitAdelete(this->func->m_alloc, this->bailInSymbols);
 }
 
-void LinearScan::GeneratorBailIn::SpillRegsForBailIn()
```

Comment:
This function was called from inside the always false condition above. Aside from introducing segfaults, analysis of the logic shows it's unnecessary.

The purpose of this function was to free RAX and RCX (or EAX and ECX) before a GeneratorBailIn as the logic will use those two registers - BUT - the way the BailIn code works it restores whatever was in RAX and RCX before the Yield/Await as its last step, so freeing them serves no purpose.